OpenLiveWriter.Install.nuspec

```diff
@@ -0,0 +1,26 @@
+﻿<?xml version="1.0"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2011/08/nuspec.xsd">
+  <metadata>
+    <id>openlivewriter</id>
+    <version>0.5.0.0</version>
+    <title>OpenLiveWriter (Install)</title>
+    <authors>See list of authors at http://openlivewriter.org/</authors>
+    <owners>Darwin Sanoy, OpenLiveWriter</owners>
+    <licenseUrl>https://github.com/OpenLiveWriter/OpenLiveWriter/blob/master/license.txt</licenseUrl>
+    <projectUrl>http://openlivewriter.org/</projectUrl>
+    <iconUrl>https://cdn.rawgit.com/csi-windowschocolatey/openlivewriter/master/app.png</iconUrl>
```

Comment:
@martinwoodward using Rawgit is actually one of the icon guidelines.  you can find them all here: https://github.com/chocolatey/choco/wiki/CreatePackages#package-icon-guidelines

Yes, ChocolateyGUI uses the listed URL in the nuspec to serve the image as well.